go-controller/hack/lint.sh

```
@@ -1,18 +1,28 @@
 #!/usr/bin/env bash
 VERSION=v1.64.8
-extra_flags=""
+: "${GOLANGCI_LINT_VERSION:=$VERSION}"
+extra_flags=(--verbose --print-resources-usage --modules-download-mode=vendor --timeout=15m0s)
 if [ "$#" -ne 1 ]; then
   if [ "$#" -eq 2 ] && [ "$2" == "fix" ]; then
-    extra_flags="--fix"
+    extra_flags+=(--fix)
   else
     echo "Expected command line argument - container runtime (docker/podman) got $# arguments: $@"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)